### PR TITLE
Allow interpolation to be empty

### LINKED ISSUE #125: Parser error on empty interpolation
The parser fails on expressions like `/#{ }/n` complaining that there should be an identifier after the `#{` . The `ruby` interpreter does not complain about this.

Several examples can be found at:
https://github.com/ruby/ruby/blob/c565dfb09ad7d55fa671f65cea7088a914bf1931/test/ruby/test_m17n.rb#L785

### PR DESCRIPTION
**Pull request by lunks**
Closes #125

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -777,7 +777,7 @@ module.exports = grammar({
     character: $ => /\?(\\\S({[0-9]*}|[0-9]*|-\S([MC]-\S)?)?|\S)/,
 
     interpolation: $ => seq(
-      '#{', $._statement, '}'
+     "#{", optional($._statement),"}"
```
Comment (aibaars) [inline grammar.js:780]: It looks like we use single quotes in this file, best keep it consistent.
```suggestion
     '#{', optional($._statement),'}'
```

**Review by aibaars — APPROVED** [review]
This looks good to me. Could you address the conflict?

@maxbrunsfeld could you merge this pull request if you are happy with the change?

File: grammar.js (hunk 1 above)
Comment (lunks) [reply grammar.js:780]: There are places where `"` are being used, but I changed given it's indeed more common.

**Review by aibaars — APPROVED** [review]

**Comment by maxbrunsfeld** [comment]
Thank you!

**Pull request merged**